jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/WellKnownLabelsEnricher.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.enricher.generic;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jkube.kit.common.Configs;
+import org.eclipse.jkube.kit.config.resource.GroupArtifactVersion;
+import org.eclipse.jkube.kit.enricher.api.JKubeEnricherContext;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+public class WellKnownLabelsEnricher extends AbstractLabelEnricher {
+  public static final String KUBERNETES_APP_LABEL = "app.kubernetes.io/";
+  private static final String WELL_KNOWN_LABELS_PROPERTY = "jkube.kubernetes.well-known-labels";
+
+  @AllArgsConstructor
+  private enum Config implements Configs.Config {
+    USE_WELL_KNOWN_LABELS("useWellKnownLabels", "true"),
```

Comment:
We should be consistent with this setting in other Enrichers.

For example, DebugEnricher uses `ENABLED` for the same purpose.

I understand that you inferred this from ProjectLabelEnricher.Config.USE_PROJECT_LABEL, but AFAIU that in this Enricher, it has a different purpose.